### PR TITLE
Feature/component modal update

### PR DESCRIPTION
**Pull request by colorful-tones**
Updates, as per requested

### CODE REVIEW

**Review by garand — APPROVED** [review]

**Comment by garand** [comment]
@timwright12 This looks good to me, could you review it when you have a moment?

File: _component/modal/_notes.md [hunk 1]
```diff
@@ -1 +1,14 @@
-The modal dialog component is a standard and accessible JavaScript plugin that was converted from a jQuery plugin by Scott Ohara, you can [view the plugin on Github](https://github.com/scottaohara/accessible_modal_window). This component is keyboard accessible, reusable, and properly traps focus while the dialog is open. It also exposes a small API via a callback function you can hook into if you need to extend the functionality. No jQuery is required.
+We highly recommend the following:
+
+[a11y-dialog](http://edenspiekermann.github.io/a11y-dialog/) is a lightweight (1.3Kb) yet flexible script to create accessible dialog windows.
+
+✔︎ No dependencies
```
Comment (timwright12) [inline _component/modal/_notes.md]: @colorful-tones @garand This is rendering out as a paragraph, should be an unordered list

File: _component/modal/_notes.md [hunk 2]
```diff
@@ -1 +1,14 @@
-The modal dialog component is a standard and accessible JavaScript plugin that was converted from a jQuery plugin by Scott Ohara, you can [view the plugin on Github](https://github.com/scottaohara/accessible_modal_window). This component is keyboard accessible, reusable, and properly traps focus while the dialog is open. It also exposes a small API via a callback function you can hook into if you need to extend the functionality. No jQuery is required.
+We highly recommend the following:
+
+[a11y-dialog](http://edenspiekermann.github.io/a11y-dialog/) is a lightweight (1.3Kb) yet flexible script to create accessible dialog windows.
```
Comment (timwright12) [inline _component/modal/_notes.md:3]: @colorful-tones @garand Can we get some more information as to exactly why we're recommending this? Are there React implementations of it as well?

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: _component/modal/_notes.md (hunk 1 above)
Comment (colorful-tones) [reply _component/modal/_notes.md]: I originally copy/pasted from `a11y-dialog` [`README.md`](https://github.com/edenspiekermann/a11y-dialog/blob/master/README.md)

I now added bullets.

File: _component/modal/_notes.md (hunk 2 above)
Comment (colorful-tones) [reply _component/modal/_notes.md:3]: I copy/pasted the "Implementations" section from `a11y-dialog` [`README.md`](https://github.com/edenspiekermann/a11y-dialog/blob/master/README.md)

@garand is there any further clarification you would like to see here please?

**Pull request merged**